src/include/miopen/solver_id.hpp

```diff
@@ -62,6 +62,7 @@ struct Id
     }
     bool operator!=(const Id& other) const { return !(*this == other); }
 
+    // For comparing with legacy find db items
```

Comment:
Why not remove it ? In this PR and following it, a value of `gemm` for solver ID should be invalid.

Reply:
It is sometimes present in old database files, so we have to ignore specific cases rather than crash on them.

Reply:
I agree with Jehandad. System databases shall be updated. The old user databases include VERSION_TWEAK (git hash) so these will not be used. I think that we do not need "gemm" solver anymore.